package.json

```diff
@@ -6,6 +6,32 @@
     "playground"
   ],
   "scripts": {
-    "watch": "npm run -w @cooklang/cooklang-ts watch & npm run -w cooklang-playground watch"
+    "playground": "wireit"
```

Comment:
@Zheoni note that I ended up changing the command to `npm run playground`. Thought that might make it clearer.